scripts/tests/run_test_suite.py

```diff
@@ -145,12 +149,18 @@ def main(context, dry_run, log_level, target, target_glob, target_skip_glob,
     elif runner == 'darwin_framework_tool_python':
         runtime = TestRunTime.DARWIN_FRAMEWORK_TOOL_PYTHON
 
-    if chip_tool is None and not runtime == TestRunTime.MATTER_REPL_PYTHON:
-        paths_finder = PathsFinder()
-        if runtime == TestRunTime.CHIP_TOOL_PYTHON:
-            chip_tool = paths_finder.get('chip-tool')
-        else:
-            chip_tool = paths_finder.get('darwin-framework-tool')
+    if chip_tool is not None:
+        chip_tool = SubprocessInfo(kind='tool', path=chip_tool)
+    else:
+        if not runtime == TestRunTime.MATTER_REPL_PYTHON:
```

Comment:
Extra indentation not needed here

```suggestion
    elif not runtime == TestRunTime.MATTER_REPL_PYTHON:
        ...
```